packages/fast-usdc/src/exos/advancer.js

```diff
@@ -161,6 +158,26 @@ export const prepareAdvancerKit = (
             const destination = chainHub.makeChainAddress(EUD);
 
             const fullAmount = toAmount(evidence.tx.amount);
+            const {
+              tx: { forwardingAddress },
+              txHash,
+            } = evidence;
+
+            const { borrowerFacet, notifyFacet, poolAccount } = this.state;
+            if (
+              notifyFacet.forwardIfMinted(
+                destination,
+                forwardingAddress,
+                fullAmount,
+                txHash,
+              )
+            ) {
+              // settlement already received; tx will Forward.
```

Comment:
It's not "will Forward" because that has started at this point. I don't think this comment adds enough over the log message and the `if (forwarded)` branch
```suggestion
```